src/backend/distributed/utils/node_metadata.c

```diff
@@ -263,6 +267,43 @@ get_shard_id_for_distribution_column(PG_FUNCTION_ARGS)
 
 
 /*
+ * For the time-being master_activate_node is a wrapper around
+ * ReplicateAllReferenceTablesToNode().
```

Comment:
No, I mean maybe we should change the comment. :) It says;

master_activate_node is a wrapper around ReplicateAllReferenceTablesToNode() but it actually wraps ActiveNode().

However since it indirectly wraps ReplicateAllReferenceTablesToNode() too, it is your call to decide what do do